01_install_requirements.sh

```diff
@@ -25,6 +25,9 @@ fi
 if ! grep -q zeromq /etc/yum.repos.d/epel.repo; then
   sudo sed -i '/enabled=1/a exclude=zeromq*' /etc/yum.repos.d/epel.repo
 fi
+if ! grep -q zmq /etc/yum.repos.d/epel.repo; then
+  sudo sed -i '/^exclude=zeromq/ s/$/,*zmq/' /etc/yum.repos.d/epel.repo
+fi
```

Comment:
This works around the problem for me, but I had to run it manually on my system.  `sudo yum -y update` from line 13 failed, so this will need to get run before that on an existing system.